src/routes/game/components/dialogs/components/FiveDialog.vue

```diff
@@ -47,14 +47,20 @@ const { player } = storeToRefs(gameStore);
 const selectedCard = ref();
 const emit = defineEmits(['resolve-five']);
 
-const disableButton = computed(() => !player.value.hand.length || !selectCard.value);
+const disableButton = computed(() => player.value.hand.length && !selectedCard.value);
 const title = computed(() => t(player.value.hand.length ? 'game.dialogs.five.discardAndDraw' : 'game.dialogs.five.nice'));
 const dialog = computed(() => t(player.value.hand.length ? 'game.dialogs.five.resolveFive' : 'game.dialogs.five.resolveFiveNoCards'));
 const buttonText = computed(() => t(player.value.hand.length ? 'game.dialogs.five.discardAndDraw' : 'rules.draw'));
 
 const selectCard = (index) => {
   selectedCard.value = player.value.hand[index].id;
 };
+
+function resolveFive() {
+  emit('resolve-five', selectedCard.value);
+  selectedCard.value = null;
```

Comment:
This is what clears out state so that the next time the FiveDialog doesn't allow you to "discard" the same card and skip discarding